.github/workflows/performance-metrics.yml

```diff
@@ -45,3 +45,32 @@ jobs:
                   TARGET: production
                   CLOUD_ORIGIN: ${{ vars.MRT_STG_CLOUD_ORIGIN }}
                   FLAGS: --wait
+
+    notify-slack:
```

Comment:
Bonus: add Slack notification, similar to what's done in another workflow: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/615e0d61aea188aa28d66a9540b0684bf6179e55/.github/workflows/e2e.yml#L271-L298